### DISCUSSION TITLE
Bug: Can not set custom "context length" in settings (maximum too large)

### DISCUSSION THREAD

**Discussion by kozuch** [open-webui/open-webui#3109] [General]
Can not set custom "context length" in settings (maximum too large) > can not select small enough value

**Steps to Reproduce:**
Settings > General > Advanced parameters > Context Length > click "Default" > ruler will open - maximal value specified in the html INPUT tag is 10240000, which is too large > can not select reasonable value (4k, 16k etc.) with ruler

=> workaround > inspect HTML and overwrite the "max" INPUT attribute value to smaller number (like 128000)

**Expected Behavior:**
Set reasonable value with ruler

**Actual Behavior:**
Value too big with ruler movement

## Environment

- **Open WebUI Version:** v0.3.1
- **Ollama (if applicable):** 0.1.43

- **Operating System:** Win11
- **Browser (if applicable):** Chrome 126.0.6478.36

**Comment by kozuch** [comment]
Is this bug not present in 0.3.4? I am not able to update to latest version so I can not check it. Why was it moved here from https://github.com/open-webui/open-webui/issues/3108?

**Reply by tjbck** [reply]
You can manually edit the input number.

**Reply by kozuch** [reply]
but the slider has wrong values, so this is a bug for me still. turn back to issue again?

**Reply by justinh-rahb** [reply]
> but the slider has wrong values, so this is a bug for me still. turn back to issue again?

No, we won't be doing that. If you need a value bigger than the slider limits, just input it manually, as suggested earlier:

<img width="69" alt="Screenshot 2024-06-14 at 8 55 22 AM" src="https://github.com/open-webui/open-webui/assets/52832301/96515388-82fe-4384-8aef-7031628804d7">

**Comment by Eormeci** [comment]
Same issue . Any solutions yet ?
